doc/man3/ERR_put_error.pod

```diff
@@ -23,6 +23,7 @@ This function is usually called by a macro.
 ERR_add_error_data() associates the concatenation of its B<num> string
 arguments with the error code added last.
 ERR_add_error_vdata() is similar except the argument is a B<va_list>.
++Multiple calls to these functions append to the current top of the error queue.
```

Comment:
there is a + character at the beginning of this line.

Reply:
Oops...

Reply:
good eyes.  fixed.  if/when merged, i hope all commits will be squashed.